ps_1.Rmd

```diff
@@ -1,17 +1,25 @@
 ---
-title: "Untitled"
+title: "Problem Set 1"
 output: rmarkdown::github_document
 ---
 
+# Question 1 
 
-# This is a section
-This is some text that is going to exist in the section. This is going to be replaced with an output.
+In this question, we're asking you to hand-build some data. Create a `data.frame` that has 30 rows and two columns. In the first column, create an ID that covers each of the rows, iterating by 3. In the second column, create a random draw from the normal distribution that has a mean that is the value in the column just to the left of it. 
 
 ```{r}
-x <- 1:10
-y <- 10:1
+d <- data.frame(
+  id = seq(from = 1, by = 3, length.out = 30)
+)
+d$var <- rnorm(n = 30, mean = d$id, sd = 5)
+```
+
+We can actually plot this to show that things are, generally working. 
 
-plot(x, y, pch = 19, main = 'A dumb plot')
+```{r}
```

Comment:
I don't really like the `with(...)` idiom. This feels unnecessary in this Problem Set.